api_v3/lib/types/EntryServerNode/base/KalturaEntryServerNode.php

```diff
@@ -0,0 +1,139 @@
+<?php
+/**
+ * @package api
+ * @subpackage objects
+ * @abstract
+ */
+abstract class KalturaEntryServerNode extends KalturaObject implements IRelatedFilterable, IApiObjectFactory
+{
+	/**
+	 * unique auto-generated identifier
+	 * @var int
+	 * @readonly
+	 */
+	public $id;
+
+	/**
+	 * @var string
+	 * @insertonly
+	 * @filter eq,in
+	 */
+	public $entryId;
+
+	/**
+	 * @var int
+	 * @readonly
+	 * @filter eq,in
+	 */
+	public $serverNodeId;
+
+	/**
+	 * @var int
+	 * @readonly
+	 */
+	public $partnerId;
+
+	/**
+	 * @var time
+	 * @readonly
+	 * @filter lte,gte,order
```

Comment:
Do we really need the filters here? 
This will be an intensive db query since we have no index on the created at.